deploy/kube-config/riemann/riemann-service.json

```diff
@@ -0,0 +1,36 @@
+{
+  "apiVersion": "v1beta3",
+  "kind": "Service",
+  "metadata": {
+    "labels": {
+      "kubernetes.io/cluster-service": "true",
+      "name": "riemann-heapster"
+    },
+    "name": "riemann-heapster"
+  },
+  "spec": {
```

Comment:
Ah! Ok. Thanks for explaining. In that case just some pointers to existing
documentation should do.

On Tue, Sep 22, 2015 at 4:00 PM, Jack Foy <notifications@github.com> wrote:

> In deploy/kube-config/riemann/riemann-service.json
> <https://github.com/kubernetes/heapster/pull/379#discussion_r40155265>:
>
> > @@ -0,0 +1,36 @@
> > +{
> > +  "apiVersion": "v1beta3",
> > +  "kind": "Service",
> > +  "metadata": {
> > +    "labels": {
> > +      "kubernetes.io/cluster-service": "true",
> > +      "name": "riemann-heapster"
> > +    },
> > +    "name": "riemann-heapster"
> > +  },
> > +  "spec": {
>
> It's a good idea, but I'm tempted to defer. We have run with riemann-dash
> in the past, and we're not currently doing so; it's a useful tool for
> exploration and certain visualizations. In practice, we stream our metrics
> from Riemann to Grafana dashboards.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/heapster/pull/379/files#r40155265>.
>